tests/io/v3/vm/test_agent_config.py

```diff
@@ -0,0 +1,55 @@
+'''
+test agent config
+'''
+import re
+
+import responses
+
+BASE_URL = 'https://cloud.tenable.com'
+
+
+@responses.activate
+def test_agent_config_details(api):
+    '''
+    Test agent_config details endpoint
+    '''
+    test_response = {
+        'auto_unlink': {'enabled': True, 'expiration': 20},
+        'software_update': False
+    }
+    agent_id: int = 1
+    responses.add(
+        responses.GET,
+        re.compile(f'{BASE_URL}/scanners/{agent_id}/agents/config'),
+        json=test_response
+    )
+    res = api.v3.vm.agent_config.details(agent_id=agent_id)
+    assert isinstance(res, dict)
+
+
+@responses.activate
+def test_agent_config_edit(api):
+    '''
+    Test agent_config edit endpoint
+    '''
+    test_response = {
+        'auto_unlink': {'enabled': True, 'expiration': 20},
+        'software_update': False
+    }
+    agent_id: int = 20
+    payload = {
+        'software_update': False,
+        'auto_unlink': {'enabled': True, 'expiration': 20}
+    }
+    responses.add(
+        responses.PUT,
+        re.compile(f'{BASE_URL}/scanners/{agent_id}/agents/config'),
```

Comment:
Please update the path as per it will be expected in v3.

Reply:
Done.